lib/nexpose/rexlite/mime/header.rb

```diff
@@ -36,7 +36,7 @@ def to_s
       end
 
       def find(idx)
-        if (idx.class == ::Fixnum)
+        if idx.kind_of?(Integer)
```

Comment:
I think these methods do the exact same thing and I have no preference on which one to use. I could not find a definitive answer on which one is "more correct" with a quick search. Since this is just a port of changes from upstream rex-mime project, I am okay ignoring hound in this case.